go.mod

```diff
@@ -0,0 +1,16 @@
+module github.com/hwsc-org/hwsc-user-svc
+
+require (
+	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
+	github.com/hwsc-org/hwsc-api-blocks v0.0.0-20190217232836-1729749fa868
+	github.com/hwsc-org/hwsc-lib v0.0.0-20190218163148-bd43cc893860
```

Comment:
We should probably start tagging and releasing `hwsc-org/hwsc-lib` at some point.

Reply:
https://github.com/hwsc-org/hwsc-lib/issues/20